esileapclient/tests/unit/osc/v1/mdc/test_mdc_offer.py

```diff
@@ -235,6 +236,7 @@ def test_mdc_offer_claim(self, mock_clouds, mock_client):
             "End Time",
             "Offer UUID",
             "Status",
+            "Purpose",
```

Comment:
However, this represents the output of a lease list, so we *do* want the Purpose field here.